csv_utility.py

```diff
@@ -0,0 +1,71 @@
+import csv
+
+class CSV:
+
+	def write_user_to_csv_file(user, file):
+		"""
+		Writes user object to a CSV file
+
+		Args- user object, filepath
+		"""
+		with open("{}".format(file), "a+", newline = "") as csv_file:
+			writer = csv.writer(csv_file)
+			writer.writerow([user.user_id, user.screen_name, user.full_name])
+
+	def get_users_from_csv_file(file):
+		"""
+		Gets information from a CSV file and returns a dictionary of users
+
+		Args- filepath
+		"""
+		with open("{}".format(file), "r", newline = "") as csv_file:
+			reader = CSV.build_user_dict(csv.reader(csv_file))
+			return reader
+
+	def build_user_dict(user_list):
```

Comment:
Since you only use this method in one place, and its implementation is tightly bound to the format of the user data (making it unlikely to be able to be used elsewhere) it could probably be inlined into the `get _users_from_csv_file` method.